EcoSoapBankTests/PickupTests.swift

```diff
@@ -7,19 +7,97 @@
 //
 
 import XCTest
+import SwiftUI
 @testable import EcoSoapBank
 
 
 class PickupTests: XCTestCase {
+    private var pickupCoordinator: PickupCoordinator!
+    private var pickupController: PickupController!
+    private var pickupProvider: MockPickupProvider!
+    private var mockDataExpectation: XCTestExpectation!
+    private var waiter: XCTWaiter!
+
+    override func setUp() {
+        super.setUp()
```

Comment:
Is this out of habit or is there a need to call super here?

Reply:
SwiftLint yelled at me. 😅